packages/manager/src/components/PlatformMaintenanceBanner/LinodePlatformMaintenanceBanner.test.tsx

```diff
@@ -0,0 +1,127 @@
+  it("doesn't render when there is no platform maintenance", async () => {
+    queryMocks.useAllAccountMaintenanceQuery.mockReturnValue({
+      data: accountMaintenanceFactory.buildList(3, {
+        type: 'reboot',
+        entity: {
+          type: 'linode',
+        },
+      }),
+    });
+
+    queryMocks.useNotificationsQuery.mockReturnValue({
+      data: [],
+    });
+
+    const { queryByText } = renderWithTheme(
+      <LinodePlatformMaintenanceBanner linodeId={1} />
+    );
+
+    expect(
+      queryByText('needs to be rebooted for critical platform maintenance.')
+    ).not.toBeInTheDocument();
+  });
+
+  it('does not render if there is a notification but not a maintenance item', async () => {
+    queryMocks.useAllAccountMaintenanceQuery.mockReturnValue({
+      data: accountMaintenanceFactory.buildList(3, {
+        type: 'reboot',
+        entity: {
+          type: 'linode',
+        },
+        reason: 'Unrelated maintenance',
+      }),
+    });
+
+    queryMocks.useNotificationsQuery.mockReturnValue({
+      data: notificationFactory.buildList(1, {
+        type: 'security_reboot_maintenance_scheduled',
+        label: 'Platform Maintenance Scheduled',
+      }),
+    });
+
+    const { queryByText } = renderWithTheme(
+      <LinodePlatformMaintenanceBanner linodeId={1} />
+    );
+
+    expect(
+      queryByText('needs to be rebooted for critical platform maintenance.')
+    ).not.toBeInTheDocument();
+  });
+
+  it('renders when a maintenance item is returned', async () => {
+    const mockPlatformMaintenance = accountMaintenanceFactory.buildList(2, {
+      type: 'reboot',
+      entity: { type: 'linode' },
+      reason: 'Your Linode needs a critical security update',
+      when: '2020-01-01T00:00:00',
+      start_time: '2020-01-01T00:00:00',
+    });
+    const mockMaintenance = [
+      ...mockPlatformMaintenance,
+      accountMaintenanceFactory.build({
+        type: 'reboot',
+        entity: { type: 'linode' },
+        reason: 'Unrelated maintenance item',
+      }),
+    ];
+
+    queryMocks.useAllAccountMaintenanceQuery.mockReturnValue({
+      data: mockMaintenance,
+    });
+
+    queryMocks.useLinodeQuery.mockReturnValue({
+      data: linodeFactory.build({
+        id: mockPlatformMaintenance[0].entity.id,
+        label: 'linode-with-platform-maintenance',
+      }),
+    });
+
+    queryMocks.useNotificationsQuery.mockReturnValue({
+      data: notificationFactory.buildList(1, {
+        type: 'security_reboot_maintenance_scheduled',
+        label: 'Platform Maintenance Scheduled',
+      }),
+    });
+
+    const { findByText } = renderWithTheme(
+      <LinodePlatformMaintenanceBanner
+        linodeId={mockPlatformMaintenance[0].entity.id}
+      />
+    );
+
+    expect(await findByText('linode-with-platform-maintenance')).toBeVisible();
+    expect(
+      await findByText((el) =>
+        el.includes('needs to be rebooted for critical platform maintenance.')
+      )
+    ).toBeVisible();
```

Comment:
I think we should remove the awaiting if we're going to statically mock. 

If we wanna make the test async, we may want to use the msw approach so there is less mocking involved

```suggestion
    expect(getByText('linode-with-platform-maintenance')).toBeVisible();
    expect(
      getByText('needs to be rebooted for critical platform maintenance.', {
        exact: false,
      })
    ).toBeVisible();
  })
```

I some tests in `PlatformMaintenanceBanner.test.tsx` also may need updating